src/raster/r.droka/r.droka.py

```diff
@@ -230,29 +232,31 @@ def main():
         )
 
         # trasforma i valori di distanza celle in valori metrici utilizzando la risoluzione raster
+        # (Transform cell distance values into metric values using raster resolution)
         grass.mapcalc("costo_m=costo*(ewres()+nsres())/2", overwrite=True)
 
         # calcola A=tangente angolo visuale (INPUT) * costo in metri
+        # (Calculate A=tangent of shadow angle (INPUT) * cost in meters)
         grass.mapcalc("A=tan($ang)*costo_m", ang=ang, overwrite=True)
         grass.mapcalc("C=$z-A", z=z, overwrite=True)
         grass.mapcalc("D=C-$dem", dem=r_elevation, overwrite=True)
         # area di espansione
```

Comment:
I couldn't find a translation that I was confident with.

Reply:
The `$ang` seems to be an input parameter, which is called shadow angle in the descriptions, https://grass.osgeo.org/grass80/manuals/addons/r.droka.html

> DESCRIPTION
> The intersection between the topographic profile and the line starting from the point of detachment with a certain angle to the horizontal line defines the point of maximum propagation (Evans and Hungr, 1993; Jaboyedoff and Labiouse, 2003). **The angle of inclination and the line of propagation are defined as zenith angle and shadow zone.** This script defines rockfall zones from a digital elevation model (DEM) and vector layer containing starting point or points.


> Shadow angle = integer
> **Angle of inclination** that defines the propagation zone. It's calculated from an horizontal plane starting from source point: highest values determine a lowest propagation.


If ever you can understand the explications around page 40/155 of this document (it is the same formula with text around), it might help to confirm. https://va.minambiente.it/File/Documento/98213